docs/_source/guides/llms/practical_guides/set_up_annotation_team.md

```diff
@@ -20,22 +20,29 @@ Here's how you can do this:
 1. Get the list of users who will be annotating:
 
 ```python
-import httpx
 import argilla as rg
 
 rg.init(
     api_url="...",
     api_key="..."
 )
 
+# get the list of users
+# optional: filter users to get only those with annotator role
+users = [u for u in rg.User.list() if u.role == "annotator"]
```

Comment:
Here we can use `UserRole.annotator`, right @alvarobartt ?

Reply:
Yes, but `UserRole` is not included as an init import so if that's the case we should `from argilla.client.sdk.users.models import UserRole`, so I guess we're good like this.

Reply:
Anyway, I guess that in Python 3.11 this may fail as we need to access Enum value https://docs.python.org/3/library/enum.html#enum.Enum.value, i.e. `users = [user for user in rg.User.list() if user.role.value == "annotator"]`

Note that the above just applies to Python 3.11 or greater

Reply:
So in that case I prefer to expose `UserRole` definition in the root module and use it.
```python
from argilla import UserRole

...
if user.role == UserRole.annotator:

```

 Also, [python refs](https://docs.argilla.io/en/docs-1.11.0-docs/reference/python/python_users.html#argilla.client.users.User.role) make reference to this `UserRole` enum. And maybe we should align also the User Model [attribute](https://docs.argilla.io/en/docs-1.11.0-docs/getting_started/installation/configurations/user_management.html#user-model)